src/main/java/hudson/plugins/git/util/BuildData.java

```diff
@@ -157,6 +157,16 @@ public Build getLastBuildOfBranch(String branch) {
         return lastBuild==null?null:lastBuild.revision;
     }
 
+    /**
+     * Gets revision author of the previous build.
+     * @return revision author of the last build.
+     *    May be null will be returned if nothing has been checked out (e.g. due to wrong repository or branch)
+     */
+    @Exported
+    public @CheckForNull String getLastBuiltRevisionAuthor() {
+        return lastBuild==null?null:lastBuild.revisionAuthor;
+    }
```

Comment:
This definition of BuidData is arbitrary imo. If you're tracking a hash you're also indirectly tracking the author associated with that has and any other commit metadata that's deemed appropriate.